src/components/organisms/Footer/Footer.jsx

```diff
@@ -1,12 +1,12 @@
 import SocialMedia from "../../molecules/SocialMedia/SocialMedia";
-import "./Footer.scss"
+import "./Footer.scss";
 
 const Footer = (props) => {
   return (
     <footer>
       <div className="contact">
-        <h1 className="contact__follow-us">{props.data.footer.followUs}</h1> 
-        <SocialMedia socialMedia = {props.data.footer.socialMedia}/>
+        <h1 className="contact__follow-us">{props.data.footer.followUs}</h1>
```

Comment:
Please send only necessary data to this component, do not send all data of the structure.